src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -24,6 +24,16 @@ public ClientAsyncAuthenticateTest()
             _log = TestLogging.GetInstance();
         }
 
+        [Fact]
+        public async Task ClientAsyncAuthenticate_SslStreamClientServerNone_UseStrongCryptoSet()
+        {
+            SslProtocols protocol = SslProtocols.None;
+            await ClientAsyncSslHelper(protocol, protocol);
+
+            // Additional manual verification.
+            // Step into the code and verify that the 'UseStrongCrypto' flag is being set.
```

Comment:
I would change 'UseStrongCrypto' to be 'SCH_USE_STRONG_CRYPTO'.  That is the real name of the SCHANNEL flag and matches the actual source code name `Interop.SspiCli.SCHANNEL_CRED.Flags.SCH_USE_STRONG_CRYPTO`

Reply:
Will fix.